tools/SwiftSyntax/SyntaxData.swift

```diff
@@ -38,6 +38,79 @@ final class SyntaxData: Equatable {
 
   let childCaches: [AtomicCache<SyntaxData>]
 
+  let positionCache: AtomicCache<AbsolutePosition>
+
+  let positionAfterSkippingLeadingTriviaCache: AtomicCache<AbsolutePosition>
```

Comment:
Hmm, I don't think `accumulateLeadingTrivia(position)` takes such time. I don't think we need to cache this.

Reply:
That's fair. I'll remove the cache.